HaxeManual/08-compiler-features.tex

```diff
@@ -173,7 +173,7 @@ \subsection{Overview}
 \label{cr-completion-interface-notes}
 
 \begin{itemize}
-	\item The position-argument can be set to 0 if the file in question contains a pipeline \ic{|} character at the position of interest. This is very useful for demonstration and testing as it allows us to ignore the byte-counting process a real IDE would have to do. The examples in this section are making use of this feature.
+	\item The position-argument can be set to 0 if the file in question contains a pipeline \ic{|} character at the position of interest. This is very useful for demonstration and testing as it allows us to ignore the byte-counting process a real IDE would have to do. The examples in this section are making use of this feature. Note that this only works after dots and opening parentheses (\ic{.|} and \ic{(|}), since everywhere else \ic{|} is treated as a bitwise or operator.
```

Comment:
Also, I don't think that's technically correct, `|` isn't valid syntax after the return type of a function for example, but position completion still doesn't work there:

```haxe
function toString():String|
{
   return "test";
}
```

`Unknown identifier : toString`